docs/application/native/guides/text-input/autofill.md

```diff
@@ -0,0 +1,365 @@
+# Autofill
+
+Autofill is a feature that allows you to fill out commonly-entered information such as email, account and address in a text input field. email, account and address in a text input field.
+Using autofill functions, you can get the retrieve the data previously entered by the user and stored or request to store data in an autofill service.
```

Comment:
Please change to:
The autofill functions is used to retrieve previously entered data by the user or request to store data in an autofill service.